src/core/ledger/ledger.test.js

```diff
@@ -784,6 +789,52 @@ describe("core/ledger/ledger", () => {
         expect(ledger.accountByName("target")).toEqual(null);
       });
     });
+
+    describe("setPayoutAddress", () => {
+      const ethAddress = parseEthAddress(
+        "0x0000000000000000000000000000000000000000"
+      );
+      const fullAddress = parseEthAddress(
+        "0xffffffffffffffffffffffffffffffffffffffff"
+      );
+      const ledger = new Ledger();
+      const id = ledger.createIdentity("USER", "Bob");
+      ledger.setPayoutAddress(id, ethAddress, fullAddress);
+      const account = ledger.accountByName("Bob");
+      if (!account) {
+        expect(account).not.toBeNull();
+        return;
+      }
```

Comment:
we don't re-use mutable state between test cases. it invites "spooky error at a distance" behavior where one test could pass while breaking a subsequent test. please refactor this so that each test has its own independent ledger instance, see other tests for examples.